modules/photon-cdn/jetpack-manifest.php

```diff
@@ -1,505 +1,534 @@
-  450 => 'modules/widget-visibility/widget-conditions/widget-conditions.css',
-  451 => 'modules/widget-visibility/widget-conditions/rtl/widget-conditions-rtl.css',
-  452 => 'modules/widget-visibility/widget-conditions/widget-conditions.js',
-  453 => 'modules/publicize/assets/publicize-rtl.css',
-  454 => 'modules/publicize/assets/publicize.min.css',
-  455 => 'modules/publicize/assets/publicize.js',
-  456 => 'modules/publicize/assets/publicize.css',
-  457 => 'modules/publicize/assets/rtl/publicize-rtl.css',
-  458 => 'modules/publicize/assets/publicize-rtl.min.css',
-  459 => 'modules/widgets/simple-payments/customizer.css',
-  460 => 'modules/widgets/simple-payments/customizer.js',
-  461 => 'modules/widgets/simple-payments/style.css',
-  462 => 'modules/widgets/twitter-timeline-admin.js',
-  463 => 'modules/widgets/facebook-likebox/style.css',
-  464 => 'modules/widgets/customizer-utils.js',
-  465 => 'modules/widgets/goodreads/css/goodreads.css',
-  466 => 'modules/widgets/goodreads/css/rtl/goodreads-rtl.css',
-  467 => 'modules/widgets/social-media-icons/style.css',
-  468 => 'modules/widgets/my-community/style.css',
-  469 => 'modules/widgets/authors/style.css',
-  470 => 'modules/widgets/social-icons/social-icons-admin.js',
-  471 => 'modules/widgets/social-icons/social-icons-admin.css',
-  472 => 'modules/widgets/social-icons/social-icons.css',
-  473 => 'modules/widgets/eu-cookie-law/eu-cookie-law.js',
-  474 => 'modules/widgets/eu-cookie-law/style.css',
-  475 => 'modules/widgets/eu-cookie-law/eu-cookie-law-admin.js',
-  476 => 'modules/widgets/flickr/style.css',
-  477 => 'modules/widgets/gallery/css/admin-rtl.min.css',
-  478 => 'modules/widgets/gallery/css/admin.css',
-  479 => 'modules/widgets/gallery/css/admin-rtl.css',
-  480 => 'modules/widgets/gallery/css/admin.min.css',
-  481 => 'modules/widgets/gallery/css/rtl/admin-rtl.css',
-  482 => 'modules/widgets/gallery/js/gallery.js',
-  483 => 'modules/widgets/gallery/js/admin.js',
-  484 => 'modules/widgets/top-posts/style.css',
-  485 => 'modules/widgets/milestone/style-admin.css',
-  486 => 'modules/widgets/milestone/milestone.js',
-  487 => 'modules/widgets/milestone/admin.js',
-  488 => 'modules/widgets/search/css/search-widget-frontend.css',
-  489 => 'modules/widgets/search/css/search-widget-admin-ui.css',
-  490 => 'modules/widgets/search/js/search-widget-admin.js',
-  491 => 'modules/widgets/search/js/search-widget.js',
-  492 => 'modules/widgets/google-plus/js/admin.js',
-  493 => 'modules/widgets/contact-info/contact-info-admin.js',
-  494 => 'modules/widgets/contact-info/contact-info-map.css',
-  495 => 'modules/widgets/image-widget/style.css',
-  496 => 'modules/widgets/customizer-controls.css',
-  497 => 'modules/widgets/google-translate/google-translate.js',
-  498 => 'modules/widgets/wordpress-post-widget/style.css',
-  499 => 'modules/widgets/gravatar-profile.css',
-  500 => 'modules/lazy-images/js/lazy-images.js',
-  501 => 'modules/wpgroho.js',
-);
+<?php 
```

Comment:
@jeherve @kraftbj do we need to upgrade this here or you'll do this during the build when packaging the release?

Reply:
It's done automatically during the release process.